sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -36,13 +40,13 @@ It supports .Net Core multi-platform projects and it can be used on non-Windows
    * [.NET Core SDK 2.0 and above](https://www.microsoft.com/net/learn/get-started/) (for .NET Core version of the scanner or if you plan to use [.NET Core Global Tool](https://www.nuget.org/packages/dotnet-sonarscanner)
 
 [[info]]
-| The flavor used to compile the Scanner for MSBuild (either .NET Framework or .NET Core) is independant of the .NET version the 
+| The flavor used to compile the Scanner for .NET (either .NET Framework or .NET Core) is independant of the .NET version the 
 | project you want to analyze has been built with. Concretely, you can analyze .NET Core code with the .NET Framework version of 
 | the Scanner. It's only relevant depending on your OS, and on the versions of .NET SDKs that are installed on your build machine.
 
 ## Installation
 
-### SonarScanner for MSBuild for .NET Framework, .NET Core and .NET
+### SonarScanner for .NET for .NET Framework, .NET Core and .NET
```

Comment:
I don't understand why .NET, .NET Framework, .NET Core. What is the purpose of ".NET"? To me there are only two ".NET Framework and .NET Core"

Reply:
Oh wow, i shoul've re-read it !